js/jsscriptfiles.js

```diff
@@ -1,10 +1,38 @@
 var names = ['Max', 'Alex', 'Rach', 'John', 'Vlad','Richard', 'Natalie', 'Craig', 'Mike', 'Brenden'];
-		
-//This loop will print all the elements from the names array
-for (index in names) { 
+
+function printNames(names){
+	for (index in names) {						
+		document.write(names[index]);
+		document.write('<br>');
+	}
+}
+
+document.write('<br>');
```

Comment:
You do not need this br anymore :) If you want to use it you have to put it in your function.